src/content-layout/styles.scss

```diff
@@ -6,60 +6,101 @@
-  grid-template-columns: minmax(0, 1fr);
-  grid-template-rows: auto awsui.$space-dark-header-overlap-distance 1fr;
+  grid-template-columns: 0 0 1fr minmax(0, var(#{custom-props.$contentLayoutMaxContentWidth})) 1fr 0 0;
+  grid-template-rows: var(#{custom-props.$contentLayoutMainGap}) min-content min-content auto awsui.$space-dark-header-overlap-distance 1fr;
   min-block-size: 100%;
 
+  @include styles.media-breakpoint-down(styles.$breakpoint-x-small) {
+    #{custom-props.$contentLayoutDefaultHorizontalPadding}: awsui.$space-l;
+  }
+
   > .background {
-    background-color: awsui.$color-background-layout-main;
-    grid-column: 1;
-    grid-row: 1 / 3;
+    grid-column: 1 / 8;
+    grid-row: 1 / 6;
     color: awsui.$color-text-body-default;
 
     &.is-overlap-disabled {
-      grid-row: 1 / 2;
+      grid-row: 1 / 5;
     }
+
+    > .header-background {
+      inline-size: 100%;
+      block-size: 100%;
+    }
+  }
+
+  > .notifications {
+    grid-column: 4;
+    grid-row: 2;
+    padding-block-end: awsui.$space-xs;
+  }
+  > .breadcrumbs {
+    grid-column: 4;
+    grid-row: 3;
+    padding-block-end: awsui.$space-xs;
+  }
+
+  &.default-padding {
+    #{custom-props.$contentLayoutMainGap}: awsui.$space-scaled-m;
+    grid-template-columns:
+      var(#{custom-props.$togglesLeftWidth}, 0) var(#{custom-props.$contentLayoutDefaultHorizontalPadding}, 0) 1fr minmax(
+        0,
+        var(#{custom-props.$contentLayoutMaxContentWidth})
+      )
+      1fr var(#{custom-props.$contentLayoutDefaultHorizontalPadding}, 0) var(#{custom-props.$togglesRightWidth}, 0);
   }
 
-  > .header {
-    grid-column: 1;
-    grid-row: 1;
+  > .header-wrapper {
+    grid-column: 4;
+    grid-row: 4;
     padding-block-end: awsui.$space-content-header-padding-bottom;
+
+    &.with-divider {
+      border-block-end: 1px solid awsui.$color-border-divider-default;
+    }
   }
 
   > .content {
-    grid-column: 1;
-    grid-row: 2 / 4;
+    grid-column: 4;
+    grid-row: 5 / 8;
   }
 
   &:not(.has-header) {
-    grid-template-rows: auto calc(awsui.$space-dark-header-overlap-distance + awsui.$space-xs + awsui.$space-scaled-xxs) 1fr;
+    grid-template-rows:
+      var(#{custom-props.$contentLayoutMainGap}) min-content min-content auto calc(
+        awsui.$space-dark-header-overlap-distance + awsui.$space-xs + awsui.$space-scaled-xxs
+      )
+      1fr;
+
     > .content {
       padding-block-start: calc(awsui.$space-xs + awsui.$space-scaled-xxs);
     }
   }
 
   &.is-overlap-disabled {
-    grid-template-rows: auto 0 1fr;
+    grid-template-rows: var(#{custom-props.$contentLayoutMainGap}) min-content min-content auto 0 1fr;
+  }
+}
+
+.layout.is-visual-refresh {
+  > .background {
+    &:not(.has-header-background) {
+      background-color: awsui.$color-background-layout-main;
+    }
+  }
+}
+
+.layout:not(.is-visual-refresh) {
+  &.has-notifications {
+    #{custom-props.$contentLayoutMainGap}: 0px;
+  }
+
+  > .notifications {
+    grid-column: 1 / 8;
```

Comment:
Why is this different in classic?

Reply:
In classic notifications always occupy the full width